openshift-hack/images/os/install.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+set -xeou pipefail
+
+yum install -y ostree rpm-ostree yum-utils selinux-policy-targeted
+curl http://base-4-10-rhel8.ocp.svc > /etc/yum.repos.d/rhel8.repo
+curl http://base-4-10-rhel8-ocp.svc/rhel-8-advanced-virt/ > /etc/yum.repos.d/rhel8-advanced-virt.repo
+curl http://base-4-10-rhel8-ocp.svc/rhel-8-fast-datapath-aarch64/ > /etc/yum.repos.d/rhel8-fast-datapath-aarch64.repo
+curl http://base-4-10-rhel8-aarch64.ocp.svc > /etc/yum.repos.d/rhel8-aarch64.repo
+
+commit=$( find /srv -name *.commit | sed -Ee 's|.*objects/(.+)/(.+)\.commit|\1\2|' | head -1 )
+mkdir /tmp/working && cd /tmp/working
+rpm-ostree db list --repo /srv/repo $commit > /tmp/packages
+
+PACKAGES=(openshift-hyperkube)
+yumdownloader -y --disablerepo=* --enablerepo=built --destdir=/tmp/rpms "${PACKAGES[@]}"
+if ! grep -q cri-o /tmp/packages; then
+  yumdownloader -y --disablerepo=* --enablerepo=rhel-8* --destdir=/tmp/rpms cri-o cri-tools
+fi
+if ! grep -q machine-config-daemon /tmp/packages; then 
+  yumdownloader -y --disablerepo=* --enablerepo=rhel-8* --destdir=/tmp/rpms machine-config-daemon
+fi
```

Comment:
maybe just use `--enablerepo=rhel-8-baseos --enablerepo=rhel-8-appstream --enablerepo=rhel-8-server-ose` to get the exact repos required